### ISSUE TITLE
Error while installing pivccu-modules-dkms

### ISSUE THREAD

**Issue by Marmelatze** [alexreinert/debmatic#34]
When running `apt install pivccu-modules-dkms` I get the following error on Debian Jessie Kernel 4.9.0-9):

```
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following NEW packages will be installed:
  pivccu-modules-dkms
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 30.2 kB of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 https://www.debmatic.de/debmatic stable/main amd64 pivccu-modules-dkms all 1.0.34 [30.2 kB]
Fetched 30.2 kB in 0s (55.8 kB/s)
Preconfiguring packages ...
Selecting previously unselected package pivccu-modules-dkms.
(Reading database ... 93312 files and directories currently installed.)
Preparing to unpack .../pivccu-modules-dkms_1.0.34_all.deb ...
Unpacking pivccu-modules-dkms (1.0.34) ...
Setting up pivccu-modules-dkms (1.0.34) ...
Usage: fixdep [-e] <depfile> <target> <cmdline>
 -e  insert extra dependencies given on stdin

Creating symlink /var/lib/dkms/pivccu/1.0.34/source ->
                 /usr/src/pivccu-1.0.34

DKMS: add completed.

Kernel preparation unnecessary for this kernel.  Skipping...

Building module:
cleaning build area...
make -j1 KERNELRELEASE=4.9.0-9-amd64 ARCH=x86 all....(bad exit status: 2)
Error! Bad return status for module build on kernel: 4.9.0-9-amd64 (x86_64)
Consult /var/lib/dkms/pivccu/1.0.34/build/make.log for more information.
Created symlink /etc/systemd/system/multi-user.target.wants/pivccu-dkms.service → /lib/systemd/system/pivccu-dkms.service.
Created symlink /etc/systemd/system/debmatic.service.wants/pivccu-dkms.service → /lib/systemd/system/pivccu-dkms.service.
Created symlink /etc/systemd/system/pivccu.service.requires/pivccu-dkms.service → /lib/systemd/system/pivccu-dkms.service.
[master 3756d69] committing changes in /etc after apt run
 4 files changed, 4 insertions(+)
 create mode 120000 systemd/system/debmatic.service.wants/pivccu-dkms.service
 create mode 120000 systemd/system/multi-user.target.wants/pivccu-dkms.service
 create mode 120000 systemd/system/pivccu.service.requires/pivccu-dkms.service
```

In /var/lib/dkms/pivccu/1.0.34/build/make.log:

```
DKMS make.log for pivccu-1.0.34 for kernel 4.9.0-9-amd64 (x86_64)
Sun Sep  1 20:34:12 CEST 2019
make -C /lib/modules/4.9.0-9-amd64/build M=/var/lib/dkms/pivccu/1.0.34/build modules
make[1]: Entering directory '/usr/src/linux-headers-4.9.0-9-amd64'
  CC [M]  /var/lib/dkms/pivccu/1.0.34/build/eq3_char_loop.o
  CC [M]  /var/lib/dkms/pivccu/1.0.34/build/plat_eq3ccu2.o
  CC [M]  /var/lib/dkms/pivccu/1.0.34/build/generic_raw_uart.o
  CC [M]  /var/lib/dkms/pivccu/1.0.34/build/pl011_raw_uart.o
  CC [M]  /var/lib/dkms/pivccu/1.0.34/build/dw_apb_raw_uart.o
  CC [M]  /var/lib/dkms/pivccu/1.0.34/build/meson_raw_uart.o
  CC [M]  /var/lib/dkms/pivccu/1.0.34/build/fake_hmrf.o
/var/lib/dkms/pivccu/1.0.34/build/fake_hmrf.c: In function ‘fake_hmrf_write’:
/var/lib/dkms/pivccu/1.0.34/build/fake_hmrf.c:266:1: warning: the frame size of 2080 bytes is larger than 2048 bytes [-Wframe-larger-than=]
 }
 ^
  CC [M]  /var/lib/dkms/pivccu/1.0.34/build/rpi_rf_mod_led.o
/var/lib/dkms/pivccu/1.0.34/build/rpi_rf_mod_led.c: In function ‘rpi_rf_mod_led_createled’:
/var/lib/dkms/pivccu/1.0.34/build/rpi_rf_mod_led.c:73:33: error: ‘LED_ON’ undeclared (first use in this function)
     led->brightness = initial ? LED_ON : LED_OFF;
                                 ^~~~~~
/var/lib/dkms/pivccu/1.0.34/build/rpi_rf_mod_led.c:73:33: note: each undeclared identifier is reported only once for each function it appears in
/usr/src/linux-headers-4.9.0-9-common/scripts/Makefile.build:315: recipe for target '/var/lib/dkms/pivccu/1.0.34/build/rpi_rf_mod_led.o' failed
make[4]: *** [/var/lib/dkms/pivccu/1.0.34/build/rpi_rf_mod_led.o] Error 1
/usr/src/linux-headers-4.9.0-9-common/Makefile:1544: recipe for target '_module_/var/lib/dkms/pivccu/1.0.34/build' failed
make[3]: *** [_module_/var/lib/dkms/pivccu/1.0.34/build] Error 2
Makefile:152: recipe for target 'sub-make' failed
make[2]: *** [sub-make] Error 2
Makefile:8: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-4.9.0-9-amd64'
Makefile:23: recipe for target 'all' failed
make: *** [all] Error 2
``` 

The kernel module is not installed:

```
$ modinfo generic_raw_uart
modinfo: ERROR: Module generic_raw_uart not found.
```

**Comment by alexreinert** [comment]
Fixed in version 1.0.35 of the package.
But you should really consider updating your system. Debian Jessie has no regular support anymore.

**Comment by Marmelatze** [comment]
Thanks now it's working again.
